### ISSUE TITLE
Need a list of general tasks that need done

### ISSUE THREAD

**Issue by ResidentBiscuit** [cpluspluscom/ChessPlusPlus#115]
To make this easier to contribute to, I think we should decide on a list of general tasks/features (user stories if you will) that need implemented. Otherwise it's going to remain pretty unorganized, with people just working on random parts of the code.

**Comment by LB--** [comment]
There's no rush, working on random things is fine in my opinion.

**Comment by ResidentBiscuit** [comment]
But it's not fine for anybody who wants to contribute. Nobody will bother if they don't even have a clue of what needs worked on and who is working on what. Some sort of organization would be a big help. Even if it's just us, it keeps us on track.

**Comment by LB--** [comment]
Isn't that what this issue tracker is?

**Comment by ResidentBiscuit** [comment]
Yes, that is what it's for, but it's currently pretty sparse and doesn't have any goals listed. I'm proposing that we decide on a list of goals that, once completed, would lead to a fairly complete game. So not just bugs and design defects, but actually moving forward with the project. Whenever the topic of this project comes up, this is always the single biggest complaint. People just don't know what actually needs worked on.

**Comment by LB--** [comment]
I don't really know myself. I've just been doing things that I personally felt needed to be done, but as this i a group project I have been trying to let other people make decisions too. I just want to be the one to work on the networking code.
